daml/Tests/Distribution/Syndication/Lifecycle.daml

```diff
@@ -144,4 +172,11 @@ preNotify parties@Parties{..} date assetId = do
   lifecycleSnapshot parties maxId
   snapshotCids <- map fst <$> queryFilter @Snapshot.Snapshot payingAgent (\s -> s.requester == payingAgent && s.deposit.asset.id.label == maxId.label && s.deposit.asset.id.version == maxId.version)
 
-  submit operator do exerciseByKeyCmd @Lifecycle.Service (operator, issuer, payingAgent) Lifecycle.SendPrePayoutSwiftMessage with assetId = maxId; exDivDate = date; ..
\ No newline at end of file
+  submit operator do exerciseByKeyCmd @Lifecycle.Service (operator, issuer, payingAgent) Lifecycle.SendPrePayoutSwiftMessage with exDivDate = date; ..
+  pure ()
+
+reporting : Party -> Party -> Bool -> Script ()
+reporting operator accountProvider isCashReporting = do 
+  depositCids <- map fst <$> query @AssetDeposit accountProvider
```

Comment:
You're not filtering the asset deposits here. Doesn't that mean that we also report bond holdings in the cash report?

Reply:
I am relying on the fact that a certain party can only be either account provider for cash or security.
Indeed, I missed out the filter
`filter (\deposit -> deposit.account.provider == accountProvider)`
as what we have in CreateReports.
Lemme add this filter.

Reply:
I think in the general case a custodian can be both cash and securities custodian. In this case they'd have to filter the deposits beforehand. I think that's acceptable for now, as we don't have a way to differentiate the two just based on the asset deposit.

Reply:
Noted and updated, thank you